### ISSUE TITLE
Error when opening buffer in directory with restricted access

### ISSUE THREAD

**Issue by seblj** [rcarriga/vim-ultest#45]
**Describe the bug**
Prints out error message `Error: vim-ultest encountered an unknown error on startup, check v:exception`

Logs could't be created as I was in the `/usr/local/share/nvim/runtime/syntax` and there isn't permission to create a file in that directory. I also tried to name the file `~/vim-ultest.log` to create it in the home directory, but that didn't work as well

**To Reproduce**
Go to `/usr/local/share/nvim/runtime/syntax` and open a buffer in neovim. It should prompt you with the error `Error: vim-ultest encountered an unknown error on startup, check v:exception`. It seems like I only get this message when opening a buffer in a directory with restricted access.

It also doesn't print the error message when running with sudo.

**Expected behavior**
Don't print any error message

**Screenshots**

https://user-images.githubusercontent.com/5160701/121495616-0f333600-c9da-11eb-9178-64cd39eb8ff3.mov

**Additional context**
At first it printed out `Error: vim-ultest remote function not detected, try running :UpdateRemotePlugins on install/update`, but this seems to be fixed by  2032c74. However, I tried to do `:UpdateRemotePlugins` again after that commit, and then it came back. I tried to do it even more times, and then it suddenly disappeared again showing the error I am receiving now.

**Comment by rcarriga** [comment]
I can't seem to be able to reproduce at all. Not sure what other info would be relevant. If you `:checkhealth` do you see any issues with the python provider?

**Comment by seblj** [comment]
It might be because I am on macOS. Don't know why this would fail only on macOS though. The python3 provider seems to be okay with `healthcheck` showing that the latest `pynvim` installed


I also tried to copy my config over on a virtual machine with ubuntu 20.04, and there it was no problem.

**Comment by rcarriga** [comment]
Sorry just to clarify, are you saying you can recreate the problem on ubuntu or not?

**Comment by seblj** [comment]
Yeah I only get this on my mac which is super weird. Exactly the same config, and I try to open a buffer in `/usr/local/share/nvim/runtime/syntax` on both OS's
